src/library/scala/sys/process/package.scala

```diff
@@ -186,7 +186,7 @@ package scala.sys {
     * }}}
     *
     * More information about the other ways of controlling I/O can be looked at
```

Comment:
Having two prepositions next to each other doesn't right to me. Perhaps, "... can be found in the Scaladoc..."